src/test/java/test/ImageTest.java

```diff
@@ -30,6 +30,7 @@ void image_tag(Product product) throws Exception {
         ));
 
         resources.getStatefulSets()
+                .filter(s -> product.getHelmReleaseName().equals(s.getName()))
```

Comment:
Because this assertion goes through all `StatefulSets`, it picked up Bibucket Mesh but that has a different image tag and failed. We can either exclude Bitbucket from these tests or filter it out this way and create a separate test for Synchrony and Bitbucket Mesh statefulsets.